pymeasure/instruments/agilent/agilent4294A.py

```diff
@@ -0,0 +1,87 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_range
+
+
+class Agilent4294A(Instrument):
+    """ Represents the Agilent 4294A Precision Impedance Analyzer """
+
+    start_frequency = Instrument.control(
+        "STAR?", "STAR %e HZ", "Set the start frequency in Hz",
+        validator=strict_range, values=[40, 140E6]
+    )
+    stop_frequency = Instrument.control(
+        "STOP?", "STOP %e HZ", "Set the stop frequency in Hz",
+        validator=strict_range, values=[40, 140E6]
```

Comment:
Just as a suggestion: It is good practice, to finish even the last element in a list etc. with a comma, such that further entries do not require to add / remove a comma in other lines.
```suggestion
        validator=strict_range, values=[40, 140E6],
```